crypto/bio/bss_bio.c

```diff
@@ -572,27 +565,34 @@ static long bio_ctrl(BIO *bio, int cmd, long num, void *ptr)
             other_b->size = b->size;
         }
 
-        ret = 1;
         break;
 
+    case BIO_CTRL_DGRAM_SET_NEXT_TIMEOUT:
```

Comment:
???  Why should the BIO pair BIO support a control that is unique to the datagram BIO

Reply:
I have practically zero knowledge here but maybe the pair and file BIOs get combined with a datagram BIO (by pushing BIOs) in some of the tests.
I had experienced some failing CI test cases where adding this `case BIO_CTRL_DGRAM_SET_NEXT_TIMEOUT` apparently solved the issue, but meanwhile I've removed the two lines you commented on here, and for whatever reason (likely due to the other fixes) all tests still pass.